locales/es-MX/eligibility.rdf.po

```diff
@@ -0,0 +1,37 @@
+msgid ""
+msgstr ""
+"Project-Id-Version: TAO 3.4.0-sprint121\n"
+"PO-Revision-Date: 2020-02-06T11:48:01\n"
+"Last-Translator: TAO Translation Team <translation@tao.lu>\n"
+"MIME-Version: 1.0\n"
+"Language: es-MX\n"
+"sourceLanguage: en-US\n"
+"targetLanguage: es-MX\n"
+"Content-Type: text/plain; charset=utf-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+
+# http://www.tao.lu/Ontologies/TAOProctor.rdf#DeliveryEligibility
+msgctxt "http://www.w3.org/2000/01/rdf-schema#label"
+msgid "Delivery Eligibility"
+msgstr ""
```

Comment:
and where are the translations?

Reply:
There are translations for the next words/strings:
* [Go](https://github.com/oat-sa/extension-tao-testcenter/blob/5434ee72955c3dd1d9bc939c62f4d731137ac26f/locales/es-MX/messages.po#L130)
* [Go to](https://github.com/oat-sa/extension-tao-testcenter/blob/5434ee72955c3dd1d9bc939c62f4d731137ac26f/locales/es-MX/messages.po#L133)
* [Monitor and manage sessions for the test site](https://github.com/oat-sa/extension-tao-testcenter/blob/5434ee72955c3dd1d9bc939c62f4d731137ac26f/locales/es-MX/messages.po#L181)
* [My Test sites](https://github.com/oat-sa/extension-tao-testcenter/blob/5434ee72955c3dd1d9bc939c62f4d731137ac26f/locales/es-MX/messages.po#L188)
* [Test centers](https://github.com/oat-sa/extension-tao-testcenter/blob/5434ee72955c3dd1d9bc939c62f4d731137ac26f/locales/es-MX/messages.po#L280)
* [Test site](https://github.com/oat-sa/extension-tao-testcenter/blob/5434ee72955c3dd1d9bc939c62f4d731137ac26f/locales/es-MX/messages.po#L287)
* [Test sites](https://github.com/oat-sa/extension-tao-testcenter/blob/5434ee72955c3dd1d9bc939c62f4d731137ac26f/locales/es-MX/messages.po#L290)

I was added `How to test` section where you can find all places with these translations.